From 0e53e7835c965e4c0d3ee5be47c67bb037808ac8 Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@digia.com>
Date: Thu, 2 Jan 2014 14:19:22 +0100
Subject: [PATCH 16/20] Extend disabling of whole-program-optimizations to MSVC
 2012 and 2013

Building with whole-program-optimizations remains a problem with MSVC
2012 and 2013, so we must continue to disable it.

Task-number: QTBUG-35835
Change-Id: Ic3131495ce0ce323ad041e51332969cc01b741c5
Reviewed-by: Simon Hausmann <simon.hausmann@digia.com>

Conflicts:
	Source/WebCore/WebCore.pri
---
 Source/WebCore/WebCore.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebCore/WebCore.pri b/Source/WebCore/WebCore.pri
index ac96de5..9f91080 100644
--- a/Source/WebCore/WebCore.pri
+++ b/Source/WebCore/WebCore.pri
@@ -317,7 +317,7 @@ win32 {
 }
 
 # Remove whole program optimizations due to miscompilations
-win32-msvc2005|win32-msvc2008|win32-msvc2010|wince*:{
+win32-msvc2005|win32-msvc2008|win32-msvc2010|win32-msvc2012|win32-msvc2013|wince*:{
     QMAKE_CFLAGS_LTCG -= -GL
     QMAKE_CXXFLAGS_LTCG -= -GL
 
-- 
1.8.5.3

